src/Compilers/Core/Portable/SpecialType.cs

```diff
@@ -263,9 +263,19 @@ public enum SpecialType : sbyte
         /// </summary>
         System_Runtime_CompilerServices_InlineArrayAttribute = 46,
 
+        /// <summary>
+        /// Indicates that the type is <see cref="System.ReadOnlySpan{T}"/> from the COR library.
+        /// </summary>
+        /// <remarks>
+        /// Check for this special type cannot be used to find the "canonical" definition of <see cref="ReadOnlySpan{T}"/>
+        /// since it is fully legal for it to come from sources other than the COR library, e.g. from `System.Memory` package.
+        /// The special type entry mostly exist so that compiler can tell this type apart when resolving other members of the COR library
+        /// </remarks>
+        System_ReadOnlySpan_T = 47,
+
         /// <summary>
         /// Count of special types. This is not a count of enum members.
```

Comment:
>Count of special types. This is not a count of enum members. [](http://example.com/codeflow?start=12&length=60)

It might make sense to add a statement explicitly saying that the value of this member is likely to change in the future as new enum members are added.